<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapAlleleDialog</class>
 <widget class="QDialog" name="MapAlleleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVAMP : Allele Frequency Map Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="30,1,1,2,1,1,1" columnstretch="1,0,0" rowminimumheight="0,0,0,0,0,0,0">
   <item row="0" column="0" colspan="3">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="6">
    <widget class="QWebView" name="webView_barchart">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="listWidget_legend"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Legends :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_exportMap">
     <property name="text">
      <string>Export Map</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_exportChart">
     <property name="text">
      <string>Export Chart</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="6">
    <widget class="QSlider" name="vSlider_bChart">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_status">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>[current status]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_snppos">
     <property name="text">
      <string>snp pos</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
